packages/aws-cdk/lib/cli-arguments.ts

```diff
@@ -14,7 +14,7 @@ export interface CliArguments {
   /**
    * The CLI command name
    */
-  readonly _: Command;
+  readonly _?: Command;
```

Comment:
needed because `cdk.json` doesn't have a specific command.

Reply:
Can we just make it two different interface then?

Reply:
If it's not CLI anymore, there's no reason to call it `_`. We only do that because of `yargs`. 

I think we are turning it into structured user input, so maybe something along the lines of `UserInput`?

Reply:
I will rename `CliArguments` to `UserInput`, and rename `_` to `command`

Reply:
https://github.com/aws/aws-cdk/pull/32821

Reply:
#32822